### PR TITLE
Add code blocks to the ch6-7 (06-using-mink and 07-contexts)

### PR DESCRIPTION
**Pull request by bocharsky-bw**
Resolve `06-using-mink` and `07-contexts` in the #12

### CODE REVIEW

**Comment by bocharsky-bw** [comment]
There's an `echo` function used in code blocks instead of `var_dump` like in video. Is it OK?

File: knpu/using-mink.md [hunk 1]
```diff
@@ -21,37 +28,52 @@ little while.
...
-the HTML of whatever page we're currently on.
+The third important object is called the page. Grab it by saying `$page = $session->getPage()`:
+
+[[[ code('24eebdad21') ]]]
+
+I want you to think of this as the JQuery object or the DOM. Anything you can do
+with JQuery - like select elements, click links and fill out fields - you can do
+with the page. Less impressively, it also knows the HTML of whatever page we're currently on.
 
 ***TIP
```
Comment (bocharsky-bw) [inline knpu/using-mink.md:65]: Does it good syntax for our markdown parser? Seems GitHub complains for it

Comment (bocharsky-bw) [inline knpu/using-mink.md:65]: Hmm, in https://github.com/knpuniversity/blog/blob/master/phpstorm-live-templates.md it works fine.

Comment (weaverryan) [inline knpu/using-mink.md:65]: great question - see the `KnpUMarkdown` class in the main repository - it's our own custom flavor of markdown :)

**Comment by weaverryan** [comment]
:+1:

**Pull request merged**